packages/readmeflow/src/04-verify.ts

```
@@ -1,77 +1,83 @@
+		const broken: string[] = [];
+		const max = Number(args["--max"]);
+		const limited = links.slice(0, Math.min(links.length, max));
+		for (const href of limited) {
+			const segment: string = href.split("#").at(0) ?? "";
+			const target = path.resolve(dir as string, segment as string);
+			try {
+				await fs.access(target);
+			} catch {
+				broken.push(href);
+			}
+		}
```

ESLint

[error] Unexpected loop, use map or reduce instead.

(functional/no-loop-statements)